LearnWithMentor.DB/dbo/Tables/UserGroup.sql

```diff
@@ -1,8 +1,9 @@
-﻿CREATE TABLE UserGroups
```

Comment:
This is a breaking change - revert table renaming

Reply:
Done